docs/docs-content/getting-started/aws/deploy-k8s-cluster.md

```diff
@@ -0,0 +1,187 @@
+host cluster.
+
+Before you proceed to next section, review the following parameters.
+
+- **Number of nodes in the pool** - This option sets the number of control plane or worker nodes in the control plane or
+  worker pool. For this tutorial, set the count to one for the control plane pool and two for the worker pool.
+
+- **Allow worker capability** - This option allows the control plane node to also accept workloads. This is useful when
+  spot instances are used as worker nodes. You can check this box if you want to.
+
+- **Instance Type** - Select the compute type for the node pool. Each instance type displays the amount of CPU, RAM, and
+  hourly cost of the instance. Select `m4.2xlarge`.
+
+- **Availability zones** - Used to specify the availability zones in which the node pool can place nodes. Select an
+  availability zone.
+
+- **Disk size** - Set the disk size to **60 GiB**.
+
+- **Instance Option** - This option allows you to choose
+  [on-demand instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-on-demand-instances.html) or
+  [spot instance](https://aws.amazon.com/ec2/spot/) for worker nodes. Select **On Demand**.
+
+![Palette clusters basic information](/getting-started/aws/getting-started_deploy-k8s-cluster_cluster_nodes_config.webp)
+
+Select **Next** to proceed with the cluster deployment.
+
+In the **Settings** section, you can configure advanced options such as when to patch the OS, enable security scans,
+manage backups, add role-based access control (RBAC) bindings, and more.
+
+For this tutorial, you can use the default settings. Click on **Validate** to continue.
+
+The **Review** section allows you to review the cluster configuration prior to deploying the cluster. Review all the
+settings and click on **Finish Configuration** to deploy the cluster.
+
+![Configuration overview of newly created AWS cluster](/getting-started/aws/getting-started_deploy-k8s-cluster_profile_cluster_profile_review.webp)
+
+Navigate to the left **Main Menu** and select **Clusters**.
+
+![Update the cluster](/getting-started/aws/getting-started_deploy-k8s-cluster_create_cluster.webp)
+
+The cluster deployment process can take 15 to 30 min. The deployment time varies depending on the cloud provider,
+cluster profile, cluster size, and the node pool configurations provided. You can learn more about the deployment
+progress by reviewing the event log. Click on the **Events** tab to view the log.
+
+![Update the cluster](/getting-started/aws/getting-started_deploy-k8s-cluster_event_log.webp)
+
+<br />
+
+While you wait for the cluster deployment process to complete, feel free to check out a video where we discuss the
+growing pains of using Kubernetes and how Palette can help your address these pain points.
+
+<br />
+
+<YouTube
+  url="https://www.youtube.com/embed/wM3hcrHbAC0"
+  title="Three Common Kubernetes Growing Pains  - and how to solve them"
+/>
```

Comment:
Does it make sense to keep this video? Is there a better resource we should use, such as diagram or other video?

Reply:
Couldn't find a better alternative and I feel there is enough content on this page at the current time. Removed.